config/.env

```diff
@@ -0,0 +1,8 @@
+# Location of WordPress core develop directory
+WP_DEVELOP_DIR=/app/public/core-git/
+
+# Location of WordPress core directory to run from
+WP_CORE_DIR=/app/public/core-git/build/
```

Comment:
Aside: Because of the `.env` bug, this environment variable won't work when starting up Lando after the first run. So `_wordpressdev_detect_core_path_relative()` will always return the `core-git/build` path.

Reply:
Hmmm, shouldn't it use the `path` in `wp-cli.yml`?

Reply:
It will, but `_wordpressdev_detect_core_path_relative` also looks at the environment variable if the `wp-cli.yml` isn't defined. But if the environment variable doesn't persist, then the environment variable method won't be reliable.

Reply:
Right, but what I mean is, if the environment variable doesn't persist, it will fall back to `wp-cli.yml`, so you could change that instead. (You'd probably wanna make sure the path in `wp-cli.yml` is in sync with what is in `.env` anyway.)